uninstaller/uninstall-helper.ps1

```diff
@@ -4,6 +4,22 @@ for ($i=0; $i -le $args.Count - 1; $i++) {
 }
 $timeStamp = $args[-1]
 
+$subfolders = Get-ChildItem "$folder\.." -Directory -ErrorAction SilentlyContinue | ForEach-Object { $_.FullName }
+
+for ($i=0; $i -le $subfolders.Count - 1; $i++) {
+  $Source = $subfolders[$i]
+  $target = $subfolders[$i]+'\..'
+  $path = "move ""$Source"" ""$target"" >nul 2>nul && echo nolock || echo lock"
+  $status = cmd.exe /c $path
+  if ($status -replace '\s','' -eq 'lock'){
+
+    Write-Host "devsuite install folder is locked by running program or file opened in editor please close all running tools/editors and try again."
```

Comment:
Update message to `DevSuite folder '$Source' is locked by running program or file opened in editor. Please close all running tools/editors and try again.` That would give a clue what specific tool should be closed.